flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/basics/StreamWindowSQLExample.java

```diff
@@ -79,16 +77,16 @@ public static void main(String[] args) throws Exception {
                         + "  COUNT(DISTINCT product) unique_products\n"
                         + "FROM orders\n"
                         + "GROUP BY TUMBLE(ts, INTERVAL '5' SECOND)";
-        Table result = tEnv.sqlQuery(query);
-        tEnv.toAppendStream(result, Row.class).print();
 
-        // after the table program is converted to DataStream program,
-        // we must use `env.execute()` to submit the job.
-        env.execute("Streaming Window SQL Job");
+        tEnv.executeSql(query).print();
+        // should output;
```

Comment:
`should output:`

Reply:
@JingsongLi - I've corrected the code based on your advice.  Please let me know you have any additional comments.  Thanks again for your review/comments.